src/Containers/RectangleTable/RectangleTable.tsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { occupancyStatusTypes } from '../CircleTable/CircleTable';
```

Comment:
This file is giving me breaking changes, but I didn't write it so I don't exactly know what the issue is here. Should I just set occupancyStatus = 'Vacant' or would an interface work better than the typing?

Reply:
Shouldn't this import just be
import typeOf { occ.. } from ...;